src/test/java/org/eclipse/che/incubator/food/FoodEndpointTest.java

```diff
@@ -28,11 +28,11 @@ public void testListAll() {
                 .then().statusCode(200)
                 .extract().as(Food[].class);
         assertEquals(5, foods.length);
-        assertEquals("Apple pie", foods[0].name);
-        assertEquals("Orange", foods[1].name);
-        assertEquals("Sandwich", foods[2].name);
-        assertEquals("Soup", foods[3].name);
-        assertEquals("Strawberry cake", foods[4].name);
+        assertEquals("Orange", foods[0].name);
+        assertEquals("Apple pie", foods[1].name);
+        assertEquals("Strawberry cake", foods[2].name);
+        assertEquals("Sandwich", foods[3].name);
+        assertEquals("Soup", foods[4].name);
```

Comment:
Thank you @svor I'm not sure why this the order changed

Maybe instead, we can sort by `name` instead of `id` for the list endpoint?
https://github.com/che-incubator/quarkus-api-example/blob/12b077d729631671f83366da6a4d243b682c8778/src/main/java/org/eclipse/che/incubator/food/FoodResource.java#L36

```
        return Food.listAll(Sort.by("name"));
```

Reply:
@dkwon17 makes sense, updated